dts/arm/st/f4/stm32f429Xi.dtsi

```diff
@@ -24,4 +25,26 @@
 			};
 		};
 	};
+
+	fmc: memory-controller@a0000000 {
+		compatible = "st,stm32-fmc";
+		reg = <0xa0000000 0x1000>;
+		clocks = <&rcc STM32_CLOCK_BUS_AHB3 0x00000001>;
+		label = "STM32_FMC";
+		status = "disabled";
+
+		fmc_sdram: fmc_sdram {
```

Comment:
these changes should go into a separate commit.

same node names as H7 should be used to be consistent.

Reply:
Fixed.